python/cuml/metrics/kl_divergence.pyx

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cupy as cp
+import numpy as np
+import cuml.internals
+from cuml.common.input_utils import determine_array_type
+from cuml.common import (input_to_cuml_array, CumlArray, logger)
+from libc.stdint cimport uintptr_t
+from cuml.raft.common.handle cimport handle_t
+from cuml.raft.common.handle import Handle
+
+cdef extern from "cuml/metrics/metrics.hpp" namespace "ML::Metrics":
+    double kl_divergence(const handle_t &handle,
+                         const double *y,
+                         const double *y_hat,
+                         int n) except +
+    float kl_divergence(const handle_t &handle,
+                        const float *y,
+                        const float *y_hat,
+                        int n) except +
+
+
+@cuml.internals.api_return_any()
+def cython_kl_divergence(P, Q, handle=None, convert_dtype=True):
```

Comment:
Why not simply naming it `kl_divergence`? It is possible to [rename Cython imports](https://stackoverflow.com/a/10121232) if they interfere.